src/ui/components/DeleteLineForm/DeleteLineForm.tsx

```diff
@@ -0,0 +1,33 @@
+import { revalidatePath } from "next/cache";
+import { SubmitButon } from "./components/SubmitButton";
+import { executeGraphQL } from "@/lib/graphql";
+import { CheckoutDeleteLinesDocument } from "@/gql/graphql";
+
+type Props = {
+	lineId: string;
+	checkoutId: string;
+};
+
+export const DeleteLineForm = ({ lineId, checkoutId }: Props) => {
+	const deleteLineAction = async (lineId: string, checkoutId: string) => {
+		"use server";
+
+		await executeGraphQL(CheckoutDeleteLinesDocument, {
+			variables: {
+				checkoutId,
+				lineIds: [lineId],
+			},
+			cache: "no-cache",
+		});
+
+		revalidatePath("/cart");
+	};
```

Comment:
There's already a function for that: `deleteLineFromCheckout` in `src/app/[channel]/(main)/cart/actions.ts`

Reply:
Or maybe I should move `DeleteLineForm` component closer to the `/cart/page.tsx`?